src/scss/cs/competition.scss

```diff
@@ -0,0 +1,51 @@
+.ioe {
+  &-row {
+    gap: 40px;
+    justify-content: center;
+  }
+
+  &-col {
+    background-color: white;
+  }
+
+  &-custom-width {
+    width: 90%;
```

Comment:
what's the intention with "custom width"? To me, this section should take up the full space of the container with the padding between the cards. It's meaning it's not as wide as the comment section below.

Reply:
The custom width is used because the columns stick to the edges of the screen otherwise. Adding the 90% trigger at medium screens and smaller fits well with the designs.

Reply:
Actually I have managed to resolve this by using padding around the div, using the full space of the container

Reply:
nice :)